assembl/graphql/__init__.py

```diff
@@ -1,4 +1,14 @@
 """The module containing all graphql schemas, mutations, views, configs"""
+from dogpile.cache import make_region
+
+visit_analytics_region = make_region().configure(
```

Comment:
On second thought, I would recommend this approach now:
- Move this setup to a new file in `assembl.lib`, called `caching.py`
- Define a function to configure the region and put this logic there.
- Use the `get_config` method to fetch the redis host, port from the config. Under test, it will take things from `testing.ini`, and under prod, `local.ini`.
- In base_env.rc, you should add a key like redis_region_visits_analytics = 120 and use this key in your config method above.
- Put an `includeme(config)` at the bottom of this new `caching.py` and call your function to create the region.
- In views/__init__.py, do a `config.include('assembl.lib.caching')`

This should be clear enough. And tests will should pick up this in the "setup_registry` method in `base_registry` fixture.